esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -551,6 +645,75 @@ static inline bool isPublishedQuery(EsdlMethodImplType implType)
     return (implType==EsdlMethodImplRoxie || implType==EsdlMethodImplWsEcl);
 }
 
+void EsdlServiceImpl::handleFeatureAuthorization(IEspContext& context, IEsdlDefMethod& esdlDef)
+{
+    auto method = esdlDef.queryName();
+    auto bindingMethodAccessMap = m_bindingAccessMaps.getValue(method);
```

Comment:
Below you make some pretty big assumptions about the type for not knowing what it is ahead of time. (auto).  And I found it hard to determine exactly what that type was.  Mostly because the type itself is owned rather than having the map just own its entries without them using an owned wrapper.

Reply:
Auto doesn't mean I don't have to know how to use the value. It simply means I don't have to know (or type) its name. Not an argument for keeping it; just a response to the "big assumptions" comment.

Reply:
Understood.  But someone reading the code should also be able to easily figure it out.  There are places where auto really makes sense or is needed.  Anywhere else, I would say error on the side of self documenting code.